### ISSUE TITLE
Accessibility Insights - failed rules

### ISSUE THREAD

**Issue by Jaqster** [microsoft/pxt-microbit#5424]
3 Accessibility Insights rules failing - 
https://webcompliance.trafficmanager.net/ScanResultsApi/GetAccessibilityScanConsolidatedReport?siteId=749ba694-9c9c-e711-80c5-0004ffc0ff2d

![image](https://github.com/microsoft/pxt-microbit/assets/22108954/a869257a-40b6-4d2b-8caa-494d21e336d6)

**Comment by abchatra** [comment]
Still failing: https://webcompliance.trafficmanager.net/ScanResultsApi/GetAccessibilityScanReport?siteId=749ba694-9c9c-e711-80c5-0004ffc0ff2d&urlId=98732
